cairo/tests/utils/args_gen.py

```diff
@@ -271,13 +281,19 @@ def _gen_arg(
         segments.load_data(struct_ptr, [instances_ptr, len(arg)])
         return struct_ptr
 
-    if arg_type_origin in (dict, ChainMap, abc.Mapping, set):
+    if arg_type_origin in (dict, ChainMap, abc.Mapping, set) or arg_type is bytearray:
```

Comment:
Is this necessary ?
`arg_type_origin = get_origin(arg_type) or arg_type`
`get_origin(bytearray) == None` but `arg_type == bytearray`